server/hedley/modules/custom/hedley_prenatal/hedley_prenatal.module

```diff
@@ -6,3 +6,66 @@
  */
 
 include_once 'hedley_prenatal.features.inc';
+
+/**
+ * Implements hook_node_presave().
+ *
+ * Migrates data from field_obstetric_history and field_obstetric_history
+ * into field_obstetric_history_step2.
+ * This logic will need to remain a couple of months, until al devices
+ * update the APP and upload their content. Can be removed around Jan 2025.
```

Comment:
Should we mark with `@todo` then for the sake of cleanup?

Reply:
@AronNovak added at #1348